src/controllers/commands/checkin.command.js

```diff
@@ -0,0 +1,2 @@
+const SAMPLE_MESSAGE = "Dummy response to a checkin \\.";
```

Comment:
Can we remove the space after "checkin"

Reply:
sure. it's just a dummy response so not consequential.

Reply:
yeah. my bad.

Reply:
no problemo